dev/Common/IsWindowsVersion.h

```diff
@@ -33,30 +33,35 @@ inline bool IsExportPresent(
 
 inline bool IsWindows10_19H1OrGreater()
 {
-    // GetPackageInfo2() added to kernelbase.dll in NTDDI_WIN10_19H1 (aka 19H1)
+    // GetPackageInfo2() added to kernelbase.dll in 19H1 (aka NTDDI_WIN10_19H1)
     return IsExportPresent(L"kernelbase.dll", "GetPackageInfo2");
```

Comment:
Here's where Gov would ask you to probe for specific features, rather than a blanket "am I on Win10 20H1"

Reply:
This is used sparingly, primarily where we need to detect downlevel behavior or implementation details that has no overt advertisement or detection mechanism. But now I'm curious. I'll do an all up review and see if some of these can be changed to more direct feature-probing-for-feature-behavior. Thanks for sparking the itch